docs/src/user-guide/struct-logs/format/syntax.md

```diff
@@ -95,4 +93,40 @@ The formatted string will be:
 2015-03-23 19:29:48.942 INFO {0} latency=56 org.apache.hadoop.metrics2.impl.MetricsConfig: loaded properties from hadoop-metrics2.properties
 ```
 
-For a list of currently supported formatters, see [Formatters](format-struct-logs-formatters).
+### Formatting kv-pair IR log events
+
+Consider the following kv-pair IR log event:
+
+:::{note}
+In the example below, for simplicity, we render the log event as JSON with the auto-generated
+kv-pairs under the `auto-generated` key, and the user-generated kv-pairs under the `user-generated`
+key, but these keys don't exist in the log event.
+:::
+
+```json
+{
+  "auto-generated": {
+    "ts": 1741371422000
+  },
+  "user-generated": {
+    "message": "Callback registered to fire in 5 seconds:",
+    "ts": 1741371427000
+  }
+}
+```
+
+We can format this using the following YScope format string:
```

Comment:
```suggestion
We can format this using the following format string:
```

Don't think "YScope" is necessary since I don't think we support any other format strings at present?